routes/keyword.js

```diff
@@ -36,10 +39,17 @@ router.get('/:word', (req, res) => {
   const word = req.params.word
   Keyword.find({ word })
     .lean()
-    .then(words => res.send(words[0].votes))
+    .then((wordList) => {
+      if (wordList.length === 0) {
+        res.status(500).send({
+          err: 'word not found',
+          givens: req.params,
+        })
+      } else res.send(wordList)
+    })
     .catch((err) => {
       res.status(500).send({
-        err: err.toString(),
+        err,
```

Comment:
But `err.toString()` provides the type of error that occurred ex. `Error: Could not find article` vs `Could not find article`, link me to where you read that it's not recommended.